modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/DDMDataProviderInvokeCommand.java

```diff
@@ -86,6 +88,22 @@ protected DDMDataProviderResponse run() throws Exception {
 		HystrixCommandGroupKey.Factory.asKey(
 			"DDMDataProviderInvokeCommandGroup");
 
+	static {
+		HystrixPlugins histrixPlugins = HystrixPlugins.getInstance();
+
+		histrixPlugins.registerPropertiesStrategy(
+			new HystrixPropertiesStrategy() {
+
+				public String getCommandPropertiesCacheKey(
+					HystrixCommandKey commandKey,
+					HystrixCommandProperties.Setter builder) {
+
+					return null;
```

Comment:
@cleydyr, can you please give more details why you've decided to return `null` here?

Thanks in advance.

Reply:
This is because returning null as the cache key is one way of circumventing the caching of properties that is causing the problem. (see [relevant code in HystrixPropertiesFactory.java](https://github.com/Netflix/Hystrix/blob/v1.5.11/hystrix-core/src/main/java/com/netflix/hystrix/strategy/properties/HystrixPropertiesFactory.java#L61-L86).